lms/envs/production.py

```diff
@@ -1074,7 +1074,7 @@ def should_show_debug_toolbar(request):
 CELERY_DEFAULT_QUEUE = AUTH_TOKENS.get("CELERY_DEFAULT_QUEUE")
 
 ### MONGO DATABASE
-LRS_MONGO_HOST = AUTH_TOKENS.get('LRS_MONGO_HOST', 'mongodb://localhost:27017')
+LRS_MONGO_HOST = AUTH_TOKENS.get('LRS_MONGO_HOST', None)
```

Comment:
You don't need to set any default here. `.get()` will automatically return `None` if the key is not found. So `LRS_MONGO_HOST = AUTH_TOKENS.get('LRS_MONGO_HOST')` should be fine.